src/core/common/inviwocore.cpp

```diff
@@ -358,9 +371,17 @@ InviwoCore::InviwoCore(InviwoApplication* app)
 
     using OptionTypes = std::tuple<unsigned int, int, size_t, float, double, std::string>;
     util::for_each_type<OptionTypes>{}(OptionStringConverterRegFunctor{}, registerPC);
+    using OptionTypes = std::tuple<unsigned int, int, size_t, float, double, std::string>;
+    util::for_each_type<OptionTypes>{}(OptionIntConverterRegFunctor{}, registerPC);
+    using OptionTypes = std::tuple<unsigned int, int, size_t, float, double, std::string>;
+    util::for_each_type<OptionTypes>{}(IntOptionConverterRegFunctor{}, registerPC);
 
     using OptionEnumTypes = std::tuple<OptionRegEnumInt, OptionRegEnumUInt>;
     util::for_each_type<OptionEnumTypes>{}(OptionStringConverterRegFunctor{}, registerPC);
+    using OptionEnumTypes = std::tuple<OptionRegEnumInt, OptionRegEnumUInt>;
```

Comment:
should not be necessary to repeat `OptionEnumTypes`

Reply:
Nope!